tenets/codelingo/effective-go/update-comment-first-word-as-subject/codelingo.yaml

```diff
@@ -0,0 +1,311 @@
+funcs:
+  - name: isNotATestFile
+    type: asserter
+    body: |
+      function(filename) {
+        // ES5 workaround
+        // TODO: don't use ES5
+        String.prototype.endsWith = String.prototype.endsWith || function(suffix) {
```

Comment:
Is this some kind of wizardry to detect whether we support ES6 or not?

Reply:
@BlakeMScurr May have made this comment. ES6 is preferable to ES5 if we want to remove hacky code such as this and replace it with more terse code